src/cmd-buildpost-ec2

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3 -u
+# An operation that mutates a build by uploading to EC2,
+# extending the meta.json with AMI information.
+
+import os,sys,json,yaml,shutil,argparse,subprocess,re,collections
+import tempfile,hashlib,gzip
+
+def run_verbose(args, **kwargs):
+    print("+ {}".format(subprocess.list2cmdline(args)))
+    subprocess.check_call(args, **kwargs)
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID",
+                    required=True)
+parser.add_argument("--region", help="EC2 region",
+                    required=True)
+parser.add_argument("--bucket", help="S3 Bucket",
+                    required=True)
+parser.add_argument("--tag", help="AMI tag in key=value format",
+                    default=[], action='append')
+args = parser.parse_args()
+
+with open('src/config/manifest.yaml') as f:
+    manifest = yaml.safe_load(f)
+
+base_name = manifest['rojig']['name']
```

Comment:
Sidenote: hmm, right yeah, that's also what we do in `cmdlib.sh`. Seems like we should allow this to be embedded somehow in the manifest itself. Although it's not clear what rpm-ostree would do with it. Or maybe just something like `$(basename $(realpath src/config/manifest.yaml) .yaml)`? Anyway, we can discuss that separately.